spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-spi/src/main/java/org/springframework/cloud/data/module/ModuleStatus.java

```diff
@@ -91,29 +92,27 @@
 	/**
 	 * Construct a new {@code ModuleStatus}.
 	 *
-	 * @param key key of the {@code ModuleDescriptor} this status is for
+	 * @param key key of the module this status is for
 	 */
-	protected ModuleStatus(ModuleDescriptor.Key key) {
+	protected ModuleStatus(ModuleKey key) {
 		this.key = key;
 	}
 
 	/**
-	 * Return the module label for the {@code ModuleDescriptor}.
+	 * Return the module label for the module.
 	 *
-	 * @see ModuleDescriptor.Key#getModuleLabel()
-	 *
-	 * @return module label for the {@code ModuleDescriptor}
+	 * @return module label for the module
 	 */
 	public String getName() {
```

Comment:
should this be getLabel() - or alternative should we simply provide a getter for the String representation of the full key?